Automattic/wp-calypso
# A4A: Implement Pressable usage section on the new Hosting page v3.

This PR includes the usage section for Pressable plans in the new hosting page v3. It displays only if the current user has a Pressable plan purchased through A4A.

**NOTE: This PR will update the production appearance, but it's acceptable since it does not alter any functionality, just the presentation.**

<img width="1366" alt="Screenshot 2024-12-16 at 6 06 17 PM" src="https://github.com/user-attachments/assets/2e25e33d-7d4e-4107-8ba3-50c26fc53080" />


Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1551

## Proposed Changes

* Include the usage section in the Pressable section only if the current agency has a plan. Additionally, the usage will not be displayed in referral mode or if the current user has a Pressable plan that was not purchased from A4A.
* **Additional**: We have now moved the logic for loading the section title from the Plan section scope. This way, we can load different text based on specific contexts (e.g. existing plans). 
* **Additional**: Not part of the main goal but a minor fix that adds a handler for the 'Request demo' button click.

## Why are these changes being made?

* This is part of the In-product hosting page v3 project.

## Testing Instructions

* Use the A4A live link and go to the `/marketplace/hosting/pressable` section.
* Make sure you purchase a Pressable plan and have some usage.
* Confirm that in the page, you see the usage stats with correct information.

## Pre-merge Checklist
